src/main/java/it/gov/pagopa/reward/repository/UserInitiativeCountersAtomicOpsRepository.java

```diff
@@ -0,0 +1,10 @@
+package it.gov.pagopa.reward.repository;
+
+import it.gov.pagopa.reward.dto.synchronous.SynchronousTransactionRequestDTO;
+import it.gov.pagopa.reward.model.counters.UserInitiativeCounters;
+import reactor.core.publisher.Mono;
+
+public interface UserInitiativeCountersAtomicOpsRepository {
+    Mono<UserInitiativeCounters> updateDate(SynchronousTransactionRequestDTO request, String initiativeId);
```

Comment:
let's use a more specific name, moreover let's write this op in order to be as possible aligned to "reward preview" code
```suggestion
    Mono<UserInitiativeCounters> findByIdThrottled(String id);
```